ci-operator/step-registry/image-based-install/ztp/preinstall-host/image-based-install-ztp-preinstall-host-commands.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
```
This file contained potentially sensitive information and has been removed.
```

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/56350/rehearse-56350-pull-ci-openshift-image-based-install-operator-main-e2e-ibio-ztp/1833080614090182656/artifacts/e2e-ibio-ztp/image-based-install-ztp-preinstall-host/build-log.txt

Try to fix that so we will be able to see the logs